app/src/main/java/com/github/se/eventradar/ui/login/LoginScreen.kt

```diff
@@ -55,57 +86,94 @@ fun LoginScreen(navigationActions: NavigationActions) {
           .setIsSmartLockEnabled(false)
           .setAvailableProviders(providers)
           .build()
-
   Column(
       modifier = Modifier.fillMaxSize().testTag("loginScreen"),
       verticalArrangement = Arrangement.Center,
       horizontalAlignment = Alignment.CenterHorizontally,
   ) {
-    Image(
-        painter = painterResource(R.drawable.image1),
-        contentDescription = "Logo",
-        modifier = Modifier.width(132.dp).height(132.dp).testTag("logo"),
-    )
-    Text(
-        text = "Welcome",
-        style =
-            TextStyle(
-                fontSize = 57.sp,
-                fontFamily = FontFamily.SansSerif,
-                fontWeight = FontWeight(400),
-                textAlign = TextAlign.Center,
-            ),
-        modifier = Modifier.testTag("loginTitle"))
+    Row(
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically) {
+          Image(
+              painter = painterResource(R.drawable.event_radar_logo),
+              contentDescription = "Logo",
+              modifier = Modifier.padding(1.dp).width(43.dp).height(68.dp).testTag("logo"),
+          )
+          Text(
+              text = "Event Radar",
+              modifier = Modifier.width(253.dp).height(55.dp).testTag("loginTitle"),
+              style =
+                  TextStyle(
+                      fontSize = 40.sp,
+                      lineHeight = 17.sp,
+                      fontFamily = FontFamily(Font(R.font.roboto)),
+                      fontWeight = FontWeight(700),
+                      color = Color(0xFF000000),
+                      textAlign = TextAlign.Center,
+                      letterSpacing = 0.25.sp,
+                  ))
+        }
+
+    Spacer(modifier = Modifier.height(240.dp))
```

Comment:
nit: it might be better to use constraints instead of just dp values to adapt to different screen sizes.

Reply:
Changed to constraints layout in latest commit.